spec/lucky/params_spec.cr

```diff
@@ -299,4 +299,21 @@ describe Lucky::Params do
       params.nested_file?(:missing).should eq({} of String => Tempfile)
     end
   end
+
+  describe "to_h" do
+    it "returns a hash for query_params" do
+      request = build_request body: "", content_type: ""
+      request.query = "filter:name=trombone&page=1&per=50"
+      params = Lucky::Params.new(request).to_h
+      params.should eq({"filter" => {"name" => "trombone"}, "page" => "1", "per" => "50"})
+    end
+
+    it "returns a hash for body_params" do
+      request = build_request body: "filter%3Aname=tuba&page=1&per=50",
+        content_type: "application/x-www-form-urlencoded"
+
+      params = Lucky::Params.new(request).to_h
+      params.should eq({"filter" => {"name" => "tuba"}, "page" => "1", "per" => "50"})
+    end
+  end
```

Comment:
It might be worth adding a spec for JSON params as well, to make sure they are work and are combined with the query params